website/blog/testing.md

```diff
@@ -0,0 +1,512 @@
+# Lower Test Toil With Better Local Testing
+
+TODO(mwhittaker): Write an intro paragraph. Describe, briefly, the three kinds
+of testing we'll look at, and explain how they increase in complexity. Also
+maybe give a quick refresher on Service Weaver and components?
+
+## 1. Unit Testing
```

Comment:
Is there anything to say about writing unit tests w/o Service Weaver  that makes it more challenging? Maybe you have to write more code to initialize multiple components, or to add integration with fakes or smthg else? 

Or it can be the fact that weavertest is a package that provide you the same experience writing unit tests and integration tests and that can run very fast / simple to write versus how you do it today?